tensorflow/tools/graph_transforms/transform_saved_model.py

```diff
@@ -0,0 +1,382 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tools for invoking the Graph Transform Tool on SavedModel files."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.core.framework import graph_pb2
+from tensorflow.python.eager import context
+from tensorflow.python.eager import function
+from tensorflow.python.framework import func_graph
+from tensorflow.python.framework import function_def_to_graph
+from tensorflow.python.framework import graph_to_function_def
+from tensorflow.python.framework import importer
+from tensorflow.python.framework import ops
+from tensorflow.python.saved_model import load
```

Comment:
@frreiss can you please check below error 

`third_party/tensorflow/tools/graph_transforms/transform_saved_model.py", line 28, in <module>
    from google3.third_party.tensorflow.python.saved_model import load
ImportError: cannot import name 'load'
-- Forge runner: Test failed`